lang/python/CMakeLists.txt

```diff
@@ -21,6 +21,23 @@ endif()
 find_package(SWIG 3.0 COMPONENTS python REQUIRED)
 include(${SWIG_USE_FILE})
 
+# Determine which wiredtiger target we should use to link against the
+# Python API. For linking purposes, we need to ensure the wiredtiger
+# library was compiled as position independent code (fPIC). This being achieved
+# either using the 'WITH_PIC' configuration option or building a shared library
+# version of WiredTiger ('ENABLE_SHARED'). Default to whichever option is available
+# (with a preference for a PIC static build), otherwise throw an error if neither
+# build is available.
+set(wiredtiger_target)
+if(ENABLE_STATIC AND WITH_PIC)
```

Comment:
If we have ENABLE_STATIC, do we also need WITH_PIC for compiling python?

Reply:
Yep, so the main use case of PIC is essentially for compiling shared libraries, where the symbols that are compiled into the shared library can be accessed with relative addresses (as shared libraries are dynamically loaded into memory). Hence any compiled object that makes up a shared library needs to be generated with PIC. 
The Python API is a shared library, hence we still need to ensure PIC is enabled if we are compiling it with a static build of WiredTiger (the static library essentially just being a collection of object files).